easybuild/easyconfigs/p/Python/Python-2.7.11-goolf-1.7.20.eb

```diff
@@ -0,0 +1,215 @@
+    ('argparse', '1.4.0', {
+        'source_urls': ['https://pypi.io/packages/source/a/argparse/'],
+    }),
+    ('pbr', '1.8.1', {
+        'source_urls': ['https://pypi.io/packages/source/p/pbr/'],
+    }),
+    ('lockfile', '0.12.2', {
+        'source_urls': ['https://pypi.io/packages/source/l/lockfile/'],
+    }),
+    ('Cython', '0.24', {
+        'source_urls': ['http://www.cython.org/release/'],
+    }),
+    ('six', '1.10.0', {
+        'source_urls': ['https://pypi.io/packages/source/s/six/'],
+    }),
+    ('dateutil', '2.5.3', {
+        'source_tmpl': 'python-%(name)s-%(version)s.tar.gz',
+        #'source_urls': ['https://pypi.io/packages/source/p/python-dateutil/'],
+        'source_urls': ['https://pypi.io/packages/3e/f5/aad82824b369332a676a90a8c0d1e608b17e740bbb6aeeebca726f17b902/'],
+    }),
+    ('deap', '1.0.2', {
+        'source_tmpl': '%(name)s-%(version)s.post2.tar.gz',
+        'source_urls': ['https://pypi.io/packages/source/d/deap/'],
+    }),
+    ('decorator', '4.0.9', {
+        'source_urls': ['https://pypi.io/packages/source/d/decorator/'],
+    }),
+    ('arff', '2.1.0', {
+        'source_tmpl': 'liac-%(name)s-%(version)s.zip',
+        'source_urls': ['https://pypi.io/packages/source/l/liac-arff/'],
+    }),
+    ('pycrypto', '2.6.1', {
+        'modulename': 'Crypto',
+        'source_urls': ['http://ftp.dlitz.net/pub/dlitz/crypto/pycrypto/'],
+    }),
+    ('ecdsa', '0.13', {
+        'source_urls': ['https://pypi.io/packages/source/e/ecdsa/'],
+    }),
+    ('paramiko', '1.17.0', {
+        'source_urls': ['https://pypi.io/packages/source/p/paramiko/'],
+    }),
+    ('pyparsing', '2.1.1', {
+        'source_urls': ['https://pypi.io/packages/source/p/pyparsing/'],
+    }),
+    ('netifaces', '0.10.4', {
+        'source_urls': ['https://pypi.io/packages/source/n/netifaces'],
+    }),
+    ('netaddr', '0.7.18', {
+        'source_urls': ['https://pypi.io/packages/source/n/netaddr'],
+    }),
+    ('funcsigs', '0.4', {
+        'source_urls': ['https://pypi.io/packages/source/f/funcsigs'],
+    }),
+    ('mock', '1.3.0', {
+        'source_urls': ['https://pypi.io/packages/source/m/mock'],
+    }),
+    ('pytz', '2015.7', {
+        'source_urls': ['https://pypi.io/packages/source/p/pytz'],
+    }),
+    ('pandas', '0.18.0', {
+        'source_urls': ['https://pypi.io/packages/source/p/pandas'],
+    }),
+    ('enum34', '1.1.2', {
+        'modulename': 'enum',
+        'source_urls': ['https://pypi.io/packages/source/e/enum34'],
+    }),
+    ('virtualenv', '14.0.6', {
+        'source_urls': ['https://pypi.io/packages/source/v/virtualenv/'],
+    }),
+    ('cycler', '0.9.0', {
+        'source_urls': ['https://pypi.io/packages/source/C/Cycler/'],
+    }),
+    ('matplotlib', '1.5.1', {
+        'source_urls': ['https://pypi.io/packages/source/m/matplotlib/'],
+    }),
+    ('ipython', '3.2.3', {
+        'source_urls': ['https://pypi.io/packages/source/i/ipython/'],
+        'modulename': 'IPython',
```

Comment:
bleh...

@pescobar is this a good idea? wouldn't it be better to stick to the (limited) list of extensions we have in the other Python 2.7.11 easyconfigs?